src/ostorlab/serve_app/oxo.py

```diff
@@ -266,6 +269,96 @@ def mutate(
             return DeleteScanMutation(result=True)
 
 
+class CreateAssetMutation(graphene.Mutation):
```

Comment:
```suggestion
class CreateAssetMutation(graphene.Mutation):
```
if this will be able to receive a list of assets it should be changed to CreateAssetsMutation